PRIVACY_MODEL.md

```diff
@@ -0,0 +1,61 @@
+This proposal introduces new metaprogramming concepts on private fields and methods. This document describes the invariants, guarantees and scope of this privacy as a whole, in light of this feature.
+
+## Private fields and methods are *hidden*
+
+Previous documentation used the term "hard private" to explain the guarantees, but that term may lead some to false conclusions. Here, we use the term *hidden*, defined as follows:
+
+> A class element is *hidden* with the syntax `#name`, which makes it inaccessible except to those granted explicit access to it.
```

Comment:
also, it makes its presence unobservable - this is important too.

Reply:
Right, will add this.